A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/flight_model.cfg

```diff
@@ -6,7 +6,7 @@ minor = 0
 max_gross_weight = 174165 ; Empty weight, (LBS)
 empty_weight = 90400 ; Empty weight, (LBS)
 reference_datum_position = 0, 0, 0 ; Position of reference datum relative to FS(0,0,0) (FEET), z, x, y
-empty_weight_CG_position = -9, 0, -1 ; Position of airplane empty weight CG relative to reference datum (FEET), z, x, y
+empty_weight_CG_position = -8.75, 0, 0 ; Position of airplane empty weight CG relative to reference datum (FEET), z, x, y
```

Comment:
Y

Reply:
Please see #3430 for full explanation. CG at empty weight must be 25% MAC. That is how airliners are designed. This is the reference line which is 18.85 meters from zero. The zero point is 2.54 meters ahead of the aircraft nose. MSFS numbers are off.

Reply:
I think you should be able to delete his comments in the changed file and mark them as resolved.